src/test/java/org/openrewrite/java/testing/assertj/JUnitAssertThrowsToAssertExceptionTypeTest.java

```diff
@@ -124,6 +122,17 @@ public void throwsExceptionWithSpecificType() {
                       });
                   }
               }
+              """,
+            """
+              import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+
+              public class SimpleExpectedExceptionTest {
+                  public void throwsExceptionWithSpecificType() {
+                      NullPointerException npe = assertThatExceptionOfType(NullPointerException.class).isThrownBy(() -> {
+                          throw new NullPointerException();
+                      }).actual();
```

Comment:
Neat! Didn't know `actual()` also worked for asserted exceptions, but it's definitely nice to see & have.

Reply:
me neither until it popped up in the IDE autocomplete 😅